src/DynamoCoreWpf/Controls/AddonsTreeView.xaml

```diff
@@ -0,0 +1,593 @@
+                                            TargetName="ExpandCollapseStateIcon"
+                                            Value="/DynamoCore;component/UI/Images/expandstate_normal.png" />
+                                </MultiDataTrigger.Setters>
+                            </MultiDataTrigger>
+
+                        </ControlTemplate.Triggers>
+                    </ControlTemplate>
+                </Setter.Value>
+            </Setter>
+
+            <Style.Triggers>
+                <DataTrigger Value="true">
+                    <DataTrigger.Binding>
+                        <Binding RelativeSource="{RelativeSource Self}"
+                                 Path="DataContext"
+                                 Converter="{StaticResource RootElementToBoolConverter}" />
+                    </DataTrigger.Binding>
+                    <Setter Property="Margin"
+                            Value="5,0,0,0" />
+                </DataTrigger>
+            </Style.Triggers>
+        </Style>
+
+        <Style x:Key="AddonsTreeViewItem"
+               TargetType="{x:Type TreeViewItem}"
+               d:DataContext="{d:DesignInstance search:BrowserItem }">
+            <Setter Property="Focusable"
+                    Value="False"></Setter>
+            <Setter Property="IsExpanded"
+                    Value="{Binding IsExpanded, Mode=OneWay}"></Setter>
+            <Setter Property="IsSelected"
+                    Value="{Binding IsSelected}"></Setter>
+            <Setter Property="Visibility">
+                <Setter.Value>
+                    <Binding Path="Visibility"
+                             Converter="{StaticResource BoolToVisibilityCollapsedConverter}" />
+                </Setter.Value>
+            </Setter>
+            <Setter Property="Template">
+                <Setter.Value>
+                    <ControlTemplate TargetType="{x:Type TreeViewItem}">
+                        <Button Command="{Binding ToggleIsExpanded}">
+                            <Button.Focusable>False</Button.Focusable>
+                            <Button.Template>
+                                <ControlTemplate TargetType="Button">
+                                    <ContentPresenter />
+                                </ControlTemplate>
+                            </Button.Template>
+
+                            <Border Name="ContainerLight"
+                                    MouseEnter="OnMemberMouseEnter"
+                                    MouseLeave="OnPopupMouseLeave">
+                                <Grid>
+                                    <Grid.RowDefinitions>
+                                        <RowDefinition Height="Auto" />
+                                        <RowDefinition Height="*" />
+                                    </Grid.RowDefinitions>
+
+                                    <Border Name="ToggleButtonAndNodeName"
+                                            Grid.Row="0">
```

Comment:
If you have ReSharper, you can attempt to remove such of these redundant attributes. For the case here, `Grid.Row="0"` is not needed (there are some below, too). I have added the following internal task for tracking purposes:

[MAGN-6171](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6171) DUI: Remove redundant XAML attributes